automatic/paint.net/paint.net.nuspec

```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
    <metadata>
     <id>paint.net</id>
-    <version>4.3.12</version>
+    <version>4.3.12.20220921</version>
```

Comment:
Please don't change the version in the pull requests created.

Our update script automatically updates the version, and manual changes may cause misbehavior.

Please revert this change.